.circleci/config.yml

```diff
@@ -189,7 +193,23 @@ jobs:
             docker login --username "${DOCKER_USER}" --password "${DOCKER_PASSWORD}"
             ./gradlew --no-daemon --parallel "-Pbranch=${CIRCLE_BRANCH}" dockerUpload
       - notify
-      
+
+  publishOpenApiSpec:
+    executor: executor_node
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "8c:7f:3d:66:05:d9:12:a2:af:58:59:bf:97:c1:c7:84"
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Build Folder
+          working_directory: .openapidoc 
+          command: |
+            npm install
```

Comment:
favour the use of `npm ci` instead of `npm install` and use cache for npm. see https://docs.google.com/document/d/1LUJ5UpJq94fCS7Hnx4Mm3efWzIVymyx3cI-2wrYMwM0

Reply:
+1